src/precompile.jl

```diff
@@ -0,0 +1,39 @@
+using PrecompileTools: @setup_workload, @compile_workload
+
+@setup_workload begin
+    # These need to be safe to call here and bake into the pkgimage, i.e. called twice.
+    Connections.__init__()
+    MultiPartParsing.__init__()
+    Parsers.__init__()
+
+    # Doesn't seem to be needed here, and might not be safe to call twice (here and during runtime)
+    # ConnectionRequest.__init__()
+
+    gzip_data(data::String) = read(GzipCompressorStream(IOBuffer(data)))
+
+    # random port in the dynamic/private range (49152–65535) which are are
+    # least likely to be used by well-known services
+    _port = 57813
+
```

Comment:
```suggestion
```

Reply:
This broke things because `_port` is an arg given to `serve!`, but surprisingly not 1.10, which I don't understand...

btw, I had tried do `listenany` without a port arg and got a method error.

Reply:
I've reverted the change. And I think the reason 1.10 didn't fail is that I think on 1.10 if coverage is requested pkgimages aren't used.

So I just made the precompile workload run whether or not pkgimages are requested, so that coverage isn't weird in tests.